mw-webapp/src/component/table/Table.tsx

```diff
@@ -15,7 +15,7 @@ import styles from "src/component/table/Table.module.scss";
 export const Table = () => {
   const [data, setData] = useState<DayReport[]>([]);
   const loadDayReports = async () => {
-    const dayReports = await getDayReports();
+    const dayReports = await DayReportDAL.getDayReports();
```

Comment:
If table component it could not use business models

Reply:
What do you mean?

Reply:
Task about refactor table cmponent to split component and logic is #60

Reply:
Bad approach, it brings more work